dynamic_env_pytest_tests/tests/services/http_gate/test_http_bearer.py

```diff
@@ -0,0 +1,140 @@
+import logging
+
+import allure
+import pytest
+from container import create_container
+from file_helper import generate_file
+from http_gate import upload_via_http_gate_curl
+from http_gw.http_utils import get_object_and_verify_hashes
+from neofs_env.neofs_env_test_base import NeofsEnvTestBase
+from python_keywords.acl import (
+    EACLAccess,
+    EACLOperation,
+    EACLRole,
+    EACLRule,
+    bearer_token_base64_from_file,
+    create_eacl,
+    form_bearertoken_file,
+    set_eacl,
+    sign_bearer,
+    wait_for_cache_expired,
+)
+from wellknown_acl import PUBLIC_ACL
+
+logger = logging.getLogger("NeoLogger")
+
+
+@pytest.mark.sanity
+@pytest.mark.http_gate
+class Test_http_bearer(NeofsEnvTestBase):
```

Comment:
Why these copies of tests? It's impossible to review, I have no idea what has changed compared to the old versions.

Reply:
That is the only easy way to do the migration gracefully, keeping both test versions(with old and new devenv) work. We already did it in the same way with previous pull requests for s3 tests and we actually have a talk about this duplications.

Reply:
Tests logic is untouched, we only updating methods that somehow rely on the old devenv objects